### DISCUSSION TITLE
Rename main branch from `develop` to... `main`

### DISCUSSION THREAD

**Discussion by MrKrisKrisu** [Traewelling/traewelling#2150] [General]
Since we currently don't employ features such as separating our primary branches into categories like `develop`, `testing` or `release` I recommend renaming our sole branch back to `main` to align with a standardized naming convention.

**Comment by MrKrisKrisu** [comment]
your opinion? @jeyemwey @HerrLevin

**Comment by NyCodeGHG** [comment]
i think thats a good idea

**Comment by jeyemwey** [comment]
We purposefully named it develop back then to express that not every commit on here is ready to roll into production. While we have way more tests right now, we still rebase WIP commits into develop which are not safe to run production with.

**Reply by MrKrisKrisu** [reply]
But we haven't another branch for set levels for this - when our code is ready for release, we currently create an release. Therefore the name of the branch is not important.

But it's a little bit weird, that we have no main or release branch.

**Reply by HerrLevin** [reply]
How about a main-branch and a github workflow pushing into main everytime we release?

**Reply by NyCodeGHG** [reply]
i don't really see why main can't be non production code, the releases already have git tags

**Reply by HerrLevin** [reply]
You're right with that. But there's one thing that I just realized: hotfixes are on the rise and I would love to be able to release a hotfix without pushing the whole develop-branch to production. Especially if I won't be able to react to possible downtime on the next day in a timely manner.

**Reply by NyCodeGHG** [reply]
branching off from a git tag is still an option
